package.json

```diff
@@ -28,7 +28,7 @@
     "test:e2e": "cd packages/astro && pnpm playwright install && pnpm run test:e2e",
     "test:e2e:match": "cd packages/astro && pnpm playwright install && pnpm run test:e2e:match",
     "benchmark": "astro-benchmark",
-    "lint": "eslint --cache .",
+    "lint": "eslint . --report-unused-disable-directives",
```

Comment:
Not strictly necessary, but I've found this to be a nice cleanup.

Reply:
I think we'd want to still keep the `--cache` too?

Reply:
Ehh I've found the cache to be ... unreliable when working with type checking. Theoretically a file's lint result could be impacted by a couple of other factors -other files it imports types from, TSConfig settings- so caching becomes difficult.

Reply:
I won't deny that I've seen `--cache` acting strange before, so I could be persuaded removing it for now if there's no rejections from others.